package.json

```diff
@@ -30,6 +30,8 @@
   },
   "dependencies": {
     "@angular/animations": "~5.0.0",
+    "@angular/cdk": "^5.0.0",
+    "@angular/material": "^5.0.0",
```

Comment:
Have we checked with @vlaurin to make sure it's ok to introduce Angular material?

Reply:
Yes, checked with @vlaurin and he is happy to use Angular material.